cmd/mattermost/commands/config.go

```diff
@@ -140,6 +149,29 @@ func configGetCmdF(command *cobra.Command, args []string) error {
 	return nil
 }
 
+func configShowCmdF(command *cobra.Command, args []string) error {
+	app, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer app.Shutdown()
+
+	// check that no arguments are given
+	err = cobra.NoArgs(command, args)
+	if err != nil {
+		return err
+	}
+
+	// set up the config object
+	// Note: app.Config() returns a pointer value so make appropriate changes
+	config := app.Config()
+
+	// pretty print
+	out := prettyPrint(configToMap(*config))
```

Comment:
nit: could avoid the temporary `out` and just directly `Printf` the result of calling `prettyPrint`.

Reply:
I didn't saw this, i will update this too.